validator/src/commands/run/args.rs

```diff
@@ -1568,11 +1567,24 @@ mod tests {
                 rpc_bootstrap_config: RpcBootstrapConfig::default(),
                 blockstore_options: BlockstoreOptions::default(),
                 json_rpc_config: JsonRpcConfig {
-                    health_check_slot_distance: 128,
-                    max_multiple_accounts: Some(100),
-                    rpc_threads: num_cpus::get(),
-                    rpc_blocking_threads: 1.max(num_cpus::get() / 4),
-                    max_request_body_size: Some(MAX_REQUEST_BODY_SIZE),
+                    health_check_slot_distance: json_rpc_config::DEFAULT_HEALTH_CHECK_SLOT_DISTANCE
+                        .parse()
+                        .unwrap(),
+                    max_multiple_accounts: Some(
+                        json_rpc_config::DEFAULT_MAX_MULTIPLE_ACCOUNTS
+                            .parse()
+                            .unwrap(),
+                    ),
+                    rpc_threads: json_rpc_config::DEFAULT_RPC_THREADS.parse().unwrap(),
+                    rpc_blocking_threads: json_rpc_config::DEFAULT_RPC_BLOCKING_THREADS
+                        .parse()
+                        .unwrap(),
+                    rpc_niceness_adj: json_rpc_config::DEFAULT_RPC_NICENESS_ADJ.parse().unwrap(),
+                    max_request_body_size: Some(
+                        json_rpc_config::DEFAULT_RPC_MAX_REQUEST_BODY_SIZE
+                            .parse()
+                            .unwrap(),
+                    ),
```

Comment:
What do you think about putting this into a test-only function within `json_rpc_config.rs` ?

Reply:
> I guess it could be a good idea. are you planning to remove pub(crate)?

Yes, I think we can 😄 

> to me, I feel like we can leave them for now and improve it later.

This goal of this PR is cleanup and this is similarly aligned cleanup. Namely, we want to keep as much logic around the JSON RPC args in the `json_rpc_confg.rs` file. I think we should set the precedent now and do a helper now so that this function doesn't get out of hand (ie 100+ line)